apps/api/src/auth.router.ts

```diff
@@ -28,7 +28,7 @@ export const authRouter = (
 
     reply.clearCookie("session", {
       path: "/",
-      domain: new URL(env.FRONTEND_URL).hostname,
+      domain: new URL(env.FRONTEND_URL as string).hostname,
```

Comment:
why do you have to use `as` here? `env` is already giving this type defined right?

No point in using typescript if you are going to force the types everywhere.